hw/ip/otbn/dv/otbnsim/sim/dmem.py

```diff
@@ -166,9 +166,9 @@ def load_i32(self, addr: int) -> int:
         idxW = idx32 // 8
         offW = idx32 % 8
 
-        uword = self.data[idxW]
-        # Extract the right 32-bit unsigned value
-        u32 = (uword >> 8 * offW) & ((1 << 32) - 1)
+        u32s = self._get_u32s(idxW)
+        u32 = u32s[7 - offW]
```

Comment:
How about making `self._get_u32s()` return the least significant 32b word first? This would be more intuitive to me, as in "words with a lower address earlier in the list".

Reply:
That's a good idea, and would get rid of both of the `7 - ...` bits, which look a bit odd (and actually another one in `_get_u32s`). I'll push up a version that does that.